target/linux/ramips/rt288x/config-4.4

```diff
@@ -119,8 +119,8 @@ CONFIG_MIPS_CMDLINE_FROM_DTB=y
 CONFIG_MIPS_L1_CACHE_SHIFT=4
 CONFIG_MIPS_L1_CACHE_SHIFT_4=y
 # CONFIG_MIPS_MACHINE is not set
-CONFIG_MIPS_NO_APPENDED_DTB=y
-# CONFIG_MIPS_RAW_APPENDED_DTB is not set
+# CONFIG_MIPS_NO_APPENDED_DTB is not set
+CONFIG_MIPS_RAW_APPENDED_DTB=y
```

Comment:
thi should be enabled when all boards are converted to using this. i do not want some boards doing it one way and other another way. either we append or use the OWRTDTB hack but now both.